integration/core/test_cli.py

```diff
@@ -802,6 +818,73 @@ def test_snapshot_purge_head_parent(bin, controller_client,
     assert info[VOLUME_HEAD]["parent"] == snap1
 
 
+def backup_volume_list(bin, controller_client, replica_client,
```

Comment:
Add a link to the issue before the function name to explain what we're testing: https://github.com/rancher/longhorn/issues/399